packages/mock-server/src/data/data.ts

```diff
@@ -747,15 +750,50 @@ export const createStocktakeLine = (
 };
 
 const createStocktakeLines = (): StocktakeLine[] => {
-  const stocktake = getStocktakes();
-  const stockLines = getStockLines();
+  const stocktakes = getStocktakes();
 
-  return stocktake
+  return stocktakes
     .map(stocktake => {
-      const stockLineSubset = takeRandomSubsetFrom(stockLines, 100);
+      const itemsSubset = takeRandomSubsetFrom(ItemData, 50);
+      return itemsSubset.map(item => createStocktakeLine(stocktake.id, item));
+    })
+    .flat();
+};
+
+const createMasterList = (n: number): MasterList => ({
+  id: faker.datatype.uuid(),
+  code: `LIST${n}`,
```

Comment:
was thinking to make some static data, in the end I didn't and made some crappy data instead.